.github/workflows/ci.yml

```diff
@@ -52,8 +52,8 @@ jobs:
     name: Unit Tests + SQL Validation
     runs-on: ubuntu-18.04
     services:
-      mysql:
-        image: mysql:latest
+      mariadb:
```

Comment:
This was a mistake with the past CI config which I only just noticed now since this PR makes bad SQL actually fail CI